workflow/snakemake_rules/fetch_sequences.smk

```diff
@@ -65,7 +65,7 @@ rule fetch_cog_uk_accessions:
     message:
         """Fetching COG-UK sample accesions (GenBank only)"""
     output:
-        cog_uk_accessions = temp("data/cog_uk_accessions.tsv")
+        cog_uk_accessions = "data/cog_uk_accessions.tsv" if config.get("keep_temp",False) else temp("data/cog_uk_accessions.tsv")
```

Comment:
If you want to keep the temp files, you can just run the snakemake command with `--notemp` flag.

Reply:
Ah that's great to know

Reply:
I've gotten rid of these constructs, thanks!